monai/bundle/scripts.py

```diff
@@ -618,3 +621,74 @@ def ckpt_export(
         more_extra_files=extra_files,
     )
     logger.info(f"exported to TorchScript file: {filepath_}.")
+
+
+def init_bundle(
+    bundle_dir: PathLike,
+    ckpt_file: Optional[PathLike] = None,
+    network: Optional[torch.nn.Module] = None,
+    metadata_str=DEFAULT_METADATA,
+    inference_str=DEFAULT_INFERENCE,
+):
+    """
+    Initialise a new bundle directory with some default configuration files and optionally network weights.
+
+    Typical usage example:
+
+    .. code-block:: bash
+
+        python -m monai.bundle init_bundle /path/to/bundle_dir network_ckpt.pt
+
+    Args:
+        bundle_dir: directory name to create, must not exist but parent direct must exist
+        ckpt_file: optional checkpoint file to copy into bundle
+        network: if given instead of ckpt_file this network's weights will be stored in bundle
+    """
+
+    bundle_dir = Path(bundle_dir).absolute()
+
+    if bundle_dir.exists():
+        raise ValueError(f"Specified bundle directory '{str(bundle_dir)}' already exists")
+
+    if not bundle_dir.parent.is_dir():
+        raise ValueError(f"Parent directory of specified bundle directory '{str(bundle_dir)}' does not exist")
+
+    configs_dir = bundle_dir / "configs"
+    models_dir = bundle_dir / "models"
+    docs_dir = bundle_dir / "docs"
+
+    bundle_dir.mkdir()
+    configs_dir.mkdir()
+    models_dir.mkdir()
+    docs_dir.mkdir()
+
+    with open(str(configs_dir / "metadata.json"), "w") as o:
```

Comment:
I've changed this to accept string or dictionary inputs for the metadata and inference scripts which would allow a caller to provide a script with their own specific formatting different from what `dump/dumps` produces.